job_definitions/code_backups.yml

```diff
@@ -57,6 +57,7 @@
       - ansicolor
     builders:
       - shell: |
+          rm -rf {{ package }}
```

Comment:
From line 47 above:

```
          wipe-workspace: false
```

Would setting this to `true` have the same effect as the `rm -rf`?

Reply:
Ah, good point, I missed that flag. I can try and set it and see how it behaves on Jenkins